### ISSUE TITLE
BackEnd Nice-to-Haves

### ISSUE THREAD

**Issue by nchislom** [GeorgeSchlosser/AdventureBookv2#9]
Admin page where game stats can be viewed, user stats, wins, execute seeding of database, and upload new story data...

**Comment by nchislom** [comment]
Began building admin components that track different game statistics (# of users, # of wins, db collection sizes, etc).

Table components are now functional within the admin-page-components branch.
All child components are nested within <AdminDiv /> React component.

Will close issue once more testing is done and branch merged with master branch

**Comment by nchislom** [comment]
<AdminDiv /> React component (and child components) are functional in branch, bringing in live data from mongodb and rendering stats and diagrams to the div.

**Comment by nchislom** [comment]
Admin console component is fully functional with db stats and dynamic story diagram. Will open next issue for FileUpload component/route to upload a new story object directly to the db.
